scripts/build.py

```diff
@@ -70,10 +70,10 @@
 
 # Files with these extensions shouldn't be moved to build directory.
 FILE_EXTENSIONS_TO_IGNORE = ('.py', '.pyc', '.stylelintrc')
-# Files with these name patterns shouldn't be moved to build directory.
-# At the point of writing this, Protractor files in /extensions share
-# the same name.
-JS_FILENAMES_TO_IGNORE = ('Spec.js', 'protractor.js')
+# Files with these name patterns shouldn't be moved to build directory, and will
+# not be served in production. (This includes protractor.js files in
+# /extensions.)
+JS_FILENAMES_SUFFIXES_TO_IGNORE = ('Spec.js', 'protractor.js')
```

Comment:
Nit: Should be JS_FILENAME_SUFFIXES_..., not JS_FILENAMES_SUFFIXES.

Reply:
Done!